test/Microsoft.ML.Benchmarks/BigramAndTrigramBenchMark.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Benchmarks
+{
+    internal class EmptyWriter : TextWriter
+    {
+        private static EmptyWriter _instance = null;
```

Comment:
nit: this could be simpified to `internal static readonly EmptyWriter  Instance = new EmptyWriter()`

+ it would be nice to add a comment with explanation why do we need that